test/e2e/components/vl-map-layer.js

```diff
@@ -14,7 +14,7 @@ class VlMapLayer extends VlElement {
   }
 
   async getFeature(id) {
-    const feature = await this.driver.executeScript(`return arguments[0]._mapElement.geoJSON.writeFeature(arguments[0].layer.getSource().getFeatures().find((feature) => feature.getId() === ${id}));`, this);
+    const feature = await this.driver.executeScript(`return arguments[0]._geoJSON.writeFeature(arguments[0].layer.getSource().getFeatures().find((feature) => feature.getId() === ${id}));`, this);
```

Comment:
hier zou je zelfs nog kunnen overwegen om ook gewoon "new GeoJson" (of wa was t daar) te doen, en zelfs die geojson ook niet op de interne API te zetten, maar het ene is in principe onlosmakelijk verbonden met het andere, dus t kan nie zo veel kwaad vermoed ik.

Reply:
en tis 'private API' ...

Reply:
OL6 heeft geen ondersteuning voor CommonJS imports wat voor problemen zorgt bij de huidige E2E setup. Het E2E object hoort bij de source code en daar is gebruik van private methodes op zich niet erg omdat het op de hoogte mag zijn van de inhoudelijke implementatie.